agent/agentssh/agentssh.go

```diff
@@ -52,6 +52,11 @@ const (
 	// MagicProcessCmdlineJetBrains is a string in a process's command line that
 	// uniquely identifies it as JetBrains software.
 	MagicProcessCmdlineJetBrains = "idea.vendor.name=JetBrains"
+
+	// BlockedFileTransferErrorCode indicates that SSH server restricted the raw command from performing
+	// the file transfer.
+	BlockedFileTransferErrorCode    = 2
```

Comment:
Is `2` a well-known code, or just one you chose?

Reply:
Actually, yes! Maybe it is better to [use code 65](https://support.microfocus.com/kb/doc.php?id=7021696).

Reply:
That documentation seems to be for https://www.microfocus.com/en-us/products/reflection-secure-it/overview ?

Reply:
You're right, my bad. There isn't any list of errors, so I'm wondering if I should stick to Error 65, as it is easier to google than "error 2" or "error 3". WDYT?

Reply:
That's reasonable 👍